### PR TITLE
Contractors guide and Golden Pandas agreement

### PR DESCRIPTION
**Pull request by michael-smith-nz**

### CODE REVIEW

**Review by ahdinosaur — APPROVED** [review]
:+1:

**Comment by danalexilewis** [comment]
good to merge. once merged ill branch and suggest some wording changes to the contractor guide. thanks michael!

as a process point i think you should merge it once you feel you have enough approval/confirmation for it.

**Pull request merged**
